### PR TITLE
Configure MTU of ENI and veths to 9001

### PR DESCRIPTION
**Pull request by hmizuma**
*Issue #, if available:*
#167 

*Description of changes:*
* Add LinkSetMTU wrapper and  mocks
* Add LinkSetMTU call to ENI setup and network test
* Change hardcoded veth MTU to 9001

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/networkutils/network.go [hunk 1]
```diff
@@ -70,6 +70,8 @@ const (
 	// - kube-proxy uses 0x0000c000
 	// - Calico uses 0xffff0000.
 	defaultConnmark = 0x80
+
+	ethernetMTU = 9001
```
Comment (liwenwu-amazon) [inline pkg/networkutils/network.go]: Can you add a comment here to indicate that 9001 also need to be defined in driver.go?

File: pkg/networkutils/network_test.go [hunk 2]
```diff
@@ -44,6 +44,7 @@ const (
 	testeniIP        = "10.10.10.20"
 	testeniMAC       = "01:23:45:67:89:ab"
 	testeniSubnet    = "10.10.0.0/16"
+	testMTU          = 9001
```
Comment (liwenwu-amazon) [inline pkg/networkutils/network_test.go]: same here, can u also add a comment that 9001 needs to be defined in network.go. thanks

File: pkg/networkutils/network.go (hunk 1 above)
Comment (hmizuma) [reply pkg/networkutils/network.go]: Updated to add comments where MTU is defined.

**Comment by xrl** [comment]
@gtaylor yes, that's how I understand this issue. Only the primary, auto-created NIC gets jumbo frames by default. This code makes it so all NICs get jumbo frames.

**Pull request merged**